src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/ReflectTypeDescriptionProvider.cs

```diff
@@ -132,6 +132,7 @@ internal ReflectTypeDescriptionProvider()
             [typeof(ICollection)] = typeof(CollectionConverter),
             [typeof(Enum)] = typeof(EnumConverter),
             [s_intrinsicNullableKey] = typeof(NullableConverter),
+            [s_intrinsicReferenceKey] = typeof(ReferenceConverter),
```

Comment:
Tests?

Reply:
I thought we already had a test that tested this table but I see the InlineData is hardcoded, my first search was poor. Will add it, thanks 😄

Reply:
Sure.  Make sure you hit this codepath: https://github.com/dotnet/runtime/blob/5c274f699d9e63d5e3054dff0b995ec186b96426/src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/ReflectTypeDescriptionProvider.cs#L1305-L1310

Reply:
Added.